bundles/org.eclipse.ui.navigator/schema/navigatorContent.exsd

```diff
@@ -385,6 +385,13 @@ If the required action provider is disabled or unbound to the current viewer, th
                </appInfo>
             </annotation>
          </attribute>
+         <attribute name="muiltiprojectEnable" type="string">
```

Comment:
The actionProvider already has an `enablement` child to configure the enablement. Can't multi-project be implemented with this?

Reply:
No, because the situation is that one or more of the selected elements has failed the enablement conditions, but at least one has passed.  The current code bails when any of the objects fails enablement conditions.  The enablement failure is legitimate for the non-Java projects selected but if there is one Java project, we can show certain menu items. We only want to do this when Projects are seletcted which is why I added the resources dependency to get IProject.  There might be another way to determine this with reflection.

Reply:
Yes, it iterates on each item in the selection.  Not sure what you mean by a core expression.  Most menus, etc.. won't be able to handle if all objects in the selection aren't enabled.  This may be a one-off case.  The Package View supports this for selecting Java projects and non-Java projects and Lars wanted the same behavior for the Project View.  I thought an optional attribute was the simplest way to do this.

Reply:
Note that `muiltiprojectEnable` has a typo and I think it would be better as more camel case  `multiProjectEnable` because there is no such thing as a `multiproject`

Reply:
Will change to camel case.

Reply:
Done